src/components/PublicHeader.spec.tsx

```diff
@@ -52,11 +52,33 @@ describe('PublicHeader', () => {
         cy.get('[data-testid="organize-button"]').should('not.exist');
     });
 
-    it('organize button leads to proper organization page', () => {
+    it('leads to organization page to pick organization', () => {
```

Comment:
Very nice, test names should start with a verb, like the sentence It (aka the component or page) 'does something'